LIBVERSION

```diff
@@ -1 +1 @@
-1.10.3
+1.11.0
```

Comment:
Is it necessary to increase the minor number? No changes in public API, isn't it?

Reply:
The two operator== in Remote.h are new.